scene/3d/collision_shape_3d.cpp

```diff
@@ -131,12 +131,18 @@ PackedStringArray CollisionShape3D::get_configuration_warnings() const {
 
 	if (shape.is_valid() && Object::cast_to<RigidBody3D>(col_object)) {
 		if (Object::cast_to<ConcavePolygonShape3D>(*shape)) {
-			warnings.push_back(RTR("ConcavePolygonShape3D doesn't support RigidBody3D in another mode than static."));
+			warnings.push_back(RTR("When used for collision, ConcavePolygonShape3D is intended to work with static CollisionShape3D nodes like StaticBody3D. \nIt will likely not behave well for or RigidBody3Ds (except when frozen and freeze_mode set to FREEZE_MODE_STATIC)."));
```

Comment:
```suggestion
			warnings.push_back(RTR("When used for collision, ConcavePolygonShape3D is intended to work with static CollisionObject3D nodes like StaticBody3D.\nIt will likely not behave well for RigidBody3Ds (except when frozen and freeze_mode set to FREEZE_MODE_STATIC)."));
```
The documentation uses the wrong type here, should be fixed along with updating that wording